src/varname.jl

```diff
@@ -266,28 +272,28 @@ Currently _not_ supported are:
   - Trailing ones: `x[2, 1]` does not subsume `x[2]` for a vector `x`
 """
 function subsumes(u::VarName, v::VarName)
-    return getsym(u) == getsym(v) && subsumes(u.lens, v.lens)
+    return getsym(u) == getsym(v) && subsumes(u.optic, v.optic)
```

Comment:
```suggestion
    return getsym(u) == getsym(v) && subsumes(getoptic(optic), getoptic(v))
```

This should have been done originally for the lens stuff, but let's just fix it now:)